src/components/WebexInMeeting/WebexInMeeting.css

```diff
@@ -0,0 +1,21 @@
+.in-meeting {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.in-meeting .remote-media {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.in-meeting .local-media {
+  position: absolute;
+  bottom: 25%;
+  right: 3%;
+  width: 20%;
+  height: 20%;
+  z-index: 1;
+  border: solid 1px black;
```

Comment:
I know I didn't make a comment about this before, but not sure we should add this border